dapp/src/pages/DashboardPage/PositionDetails.tsx

```diff
@@ -41,6 +42,10 @@ export default function PositionDetails() {
   const openDepositModal = useTransactionModal(ACTION_FLOW_TYPES.STAKE)
   const openWithdrawModal = useTransactionModal(ACTION_FLOW_TYPES.UNSTAKE)
   const activitiesCount = useAllActivitiesCount()
+  const isMobileMode = useMobileMode()
+  const arrivingSoonLabel = isMobileMode
+    ? "Arriving Soon to mobile devices"
```

Comment:
How about we add more info?
One label: `This option is not available on mobile yet. Please use the desktop app to deposit or withdraw.`

Or separate labels:
Deposit: `This option is not available on mobile yet. Please use the desktop app to deposit.`
Withdraw: `This option is not available on mobile yet. Please use the desktop app to withdraw.`

Reply:
https://github.com/thesis/acre/pull/882/commits/15bac3d35ab3932a886ce3430b8a9d060d83a573